examples/remix-auth/form/README.md

```diff
@@ -0,0 +1,17 @@
+# Remix Auth - FormStrategy
+
+This adds an example using [Remix Auth](https://github.com/sergiodxa/remix-auth) with the [FormStrategy](https://github.com/sergiodxa/remix-auth-form) to do authentication with an email and password.
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth/form)
```

Comment:
Would it make more sense for the directory to be called "email and password"?

```suggestion
[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth/email-and-password)
```

Reply:
The strategy is called FormStrategy, you don't necessarily need to use email and password, you could use it to also do username + password, or maybe add a third field.

Reply:
Ah, gotcha 👍